common/lib/capa/capa/tests/test_responsetypes.py

```diff
@@ -946,6 +946,14 @@ def gimme_a_random_hint(answer_ids, student_answers, new_cmap, old_cmap):
         hint = correct_map.get_hint('1_2_1')
         self.assertEqual(hint, self._get_random_number_result(problem.seed))
 
+    def test_empty_answer(self):
+        """
+        This Method checks if question is graded for empty answers if correct answer
+        is also not given. Fails if empty answer is scored.
+        """
```

Comment:
The doc string sounds confusing, can you update this with `Tests that problem should be graded incorrect if blank space is chosen as answer`